### PR TITLE
AWP-325: Update doc with AWP support for Plesk.

### PR DESCRIPTION
**Pull request by MariaMozgunova**
respective change to Shared Pro license docs https://github.com/cloudlinux/cloudlinux-doc/pull/227

### CODE REVIEW

File: docs/manager/README.md [hunk 1]
```diff
@@ -562,6 +562,7 @@ Check [CLI documentation](/manager/#x-ray-autotracing-cli) for a description of
 #### Requirements
 
 * CloudLinux OS Shared Pro or CloudLinux OS Solo
+* cPanel or Plesk Control Panel
```
Comment (dar-ushka) [inline docs/manager/README.md]: This part of the docs relates to X-Ray Autotracing feature, not AccelerateWP.
X-Ray and Autotracing are supported for all panels, including panel integration case.
Please, remove this line.

File: docs/manager/README.md [hunk 2]
```diff
@@ -1049,21 +1050,12 @@ In case if AccelerateWP Premium is active, the auxiliary monitoring daemon `clwp
...
-* [AccelerateWP for Plesk](https://features.cloudlinux.com/c/13-acceleratewp-for-plesk?utm_medium=social&utm_source=portal_share)
-* [AccelerateWP for DirectAdmin](https://features.cloudlinux.com/c/15-acceleratewp-for-directadmin?utm_medium=social&utm_source=portal_share)
-
-#### Does WP Accelerate tool support the DirectAdmin control panel? Do you plan to integrate it with Plesk?
-Currently AccelerateWP is compatible with cPanel only.
-
-If you are interested in supporting Plesk or DirectAdmin control panels, please record your demand on our feature portal respectively:
-* [AccelerateWP for Plesk](https://features.cloudlinux.com/c/13-acceleratewp-for-plesk?utm_medium=social&utm_source=portal_share)
-* [AccelerateWP for DirectAdmin](https://features.cloudlinux.com/c/15-acceleratewp-for-directadmin?utm_medium=social&utm_source=portal_share)
+If you want DirectAdmin to be supported, please submit your request to our feature portal [AccelerateWP for DirectAdmin](https://features.cloudlinux.com/c/15-acceleratewp-for-directadmin?utm_medium=social&utm_source=portal_share).
```
Comment (dar-ushka) [inline docs/manager/README.md]: Please, leave the previous text as is, just remove Plesk info.
As far as I can remember, the text was written and approved by Dennis.

File: docs/manager/README.md [hunk 3]
```diff
@@ -1049,21 +1050,12 @@ In case if AccelerateWP Premium is active, the auxiliary monitoring daemon `clwp
...
 
 #### Is it possible to use AccelerateWP on a non-control panel server?
-No, AccelerateWP is compatible with cPanel only.
+No, AccelerateWP is compatible only with cPanel and Plesk.
```
Comment (dar-ushka) [inline docs/manager/README.md]: Suggest to remove "only" from here, it looks redundant with two panels supported.

File: docs/manager/README.md [hunk 4]
```diff
@@ -1122,7 +1114,7 @@ All errors will be displayed in standard output and logged into `/var/log/clwpos
     rpm -q ea-php74-php-redis
     yum install ea-php74-php-redis
     ```
-8. Check Redis `ini` file is present by running the following command:
+2. Check Redis `ini` file is present by running the following command (example for cPanel):
```
Comment (dar-ushka) [inline docs/manager/README.md]: The whole section is called "Ensuring the EA-PHP Redis extension is configured correctly", thus already telling that it is cPanel-specific (ea-php packages are for cPanel only).
So, this note looks redundant.

File: docs/manager/README.md [hunk 5]
```diff
@@ -1145,20 +1137,22 @@ All errors will be displayed in standard output and logged into `/var/log/clwpos
...
-    ls /opt/alt/php74/usr/lib64/php/modules | grep redis.so
+    # ls /opt/alt/php74/usr/lib64/php/modules | grep redis.so
     redis.so
     ```
-6. If the Redis module is missing:
-   7. install the `alt-phpXY-pecl-ext` package manually
-   8. run the Redis configuration script `/usr/share/cloudlinux/wpos/enable_redis_for_alt_php.py` 
+2. If the Redis module is missing:
+   
+   Install the `alt-phpXY-pecl-ext` package manually
```
Comment (dar-ushka) [inline docs/manager/README.md]: These are steps (install, then run), could you please turn them into list, either a numbered one, or a bulleted one?

File: docs/manager/README.md [hunk 6]
```diff
@@ -1145,20 +1137,22 @@ All errors will be displayed in standard output and logged into `/var/log/clwpos
...
+   Run the Redis configuration script `/usr/share/cloudlinux/wpos/enable_redis_for_alt_php.py` 
       
       All errors will be displayed in standard output and logged into `/var/log/clwpos/main.log`.
-10. If the Redis module is present for a particular php version, but the incompatibility issue persists, re-check the following:
-    11. make sure the Redis module is loaded under user:
+3. If the Redis module is present for a particular php version, but the incompatibility issue persists, re-check the following:
+    
+    Make sure the Redis module is loaded under user:
```
Comment (dar-ushka) [inline docs/manager/README.md]: The same as above, make steps, please.

**Review by dar-ushka — COMMENTED** [review]
Check comments, please.

File: docs/manager/README.md (hunk 1 above)
Comment (MariaMozgunova) [reply docs/manager/README.md]: done

File: docs/manager/README.md (hunk 2 above)
Comment (MariaMozgunova) [reply docs/manager/README.md]: thank you

File: docs/manager/README.md (hunk 4 above)
Comment (MariaMozgunova) [reply docs/manager/README.md]: removed

File: docs/manager/README.md (hunk 5 above)
Comment (MariaMozgunova) [reply docs/manager/README.md]: done

File: docs/manager/README.md (hunk 6 above)
Comment (MariaMozgunova) [reply docs/manager/README.md]: done

**Review by dar-ushka — APPROVED** [review]

**Pull request merged**